### ISSUE TITLE
stmhal: USB_VCP when used as a stream is always in non-blocking mode, this should be documented

### ISSUE THREAD

**Issue by pfalcon** [micropython/micropython#1530]
At least documented, if not allow setting timeout.

~~~~
STATIC mp_uint_t pyb_usb_vcp_read(mp_obj_t self_in, void *buf, mp_uint_t size, int *errcode) {
    int ret = USBD_CDC_Rx((byte*)buf, size, 0);
~~~~

0 is timeout parameter.

**Comment by pfalcon** [comment]
Addressed in b7ab70c71c3db3fd81aeadb50874615977b1eadb.

**Comment by pfalcon** [comment]
Btw, for USB_VCP, it's possible to detect disconnect, and perhaps in this case it should return empty bytes. (Well, it definitely should return it, perhaps we should care to implement that.)
